src/analysis/topology/conservedMoieties/identifyConservedMoieties.m

```diff
@@ -59,55 +59,44 @@
 
 clear ATN
 
-[nMets] = size(S,1);
+[nMets] = size(N,1);
 [nAtoms] = size(A,1);
 
 xt = 1:size(A,2);
 
-% Convert incidence matrix to adjacency matrix
-[row1,col1] = find(A == 1);
-[row2,col2] = find(A == -1);
-adj = sparse([row1;row2],[row2;row1],ones(size([row1;row2]))); % Convert incidence matrix to adjacency matrix
-
-% Find connected components. Each component corresponds to an "atom conservation relation".
-if license('test','Bioinformatics_Toolbox')
-    [nComps,a2c] = graphconncomp(adj,'DIRECTED',false);
-    components = cell(nComps,1);
-    for i = 1:nComps
-        components{i} = find(a2c == i);
-    end
-else
-    % components = connectedComponents(adj); % Matlab implementation of Tarjan's algorithm. Does not work for large networks due to Matlab limitations on stack size.
-    components = find_conn_comp(adj);
-    nComps = length(components);
-end
+% Find connected components of underlying undirected graph.
+% Each component corresponds to an "atom conservation relation".
+[h,~] = find(A == -1);
+[t,~] = find(A == 1);
+adj = sparse([t;h],[h;t],ones(size([t;h]))); % Convert incidence matrix to adjacency matrix
+components = conncomp(graph(adj));
```

Comment:
Just a comment:
This function was only introduced in Matlab 2015b and wont work with previous versions.